### DISCUSSION TITLE
Welcome to exocharts-desktop Discussions!

### DISCUSSION THREAD

**Discussion by Exocharts** [Exocharts/exocharts-desktop#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Bubenchuk** [comment]
Do you plan in the future to make an API to the account to manage the account without going to the site?
